pyiron/atomistics/structure/atoms.py

```diff
@@ -1710,6 +1741,32 @@ def plot3d(
 
         view.camera = camera
         view.background = background
+
+        if camera_axis != "x" and camera_axis != "y" and camera_axis != "z":
```

Comment:
Instead of a warning it might be better to raise a value error in the else part (line 1767).